src/sentry/notifications/manager.py

```diff
@@ -300,28 +301,28 @@ def get_for_recipient_by_parent(
     def filter_to_subscribed_users(
```

Comment:
Can you change the name and docstring of this function since it accepts teams now? I'm not sure "subscribed" is a good word to use for teams since teams don't subscribe to anything ... 🤔 maybe `filter_to_accepting_recipients` or something. That's not the best but something along those lines.